### PR TITLE
add chat badges to readme

### PR DESCRIPTION
**Pull request by hrhino**
IRC, Discord, and Gitter.

~For this @emilypi or @edmundnoble or some discord admin needs to turn on the widget.~ thx based mle

~Also, the discord invite link says "haro has invited you to join scalaz". I don't know if there's a way to make it not mention me. If not, I should make my discord image match my github one.~

[It looks like this](https://github.com/hrhino/scalaz/tree/topic/chats).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ It defines a set of foundational type classes (e.g. `Functor`, `Monad`) and corr
 a large number of data structures.
 
 [![Build Status](https://secure.travis-ci.org/scalaz/scalaz.png?branch=series/7.3.x)](https://travis-ci.org/scalaz/scalaz)
+[![IRC](https://img.shields.io/badge/chat-on%20freenode-brightgreen.svg)](irc://irc.freenode.net/scalaz)
```
Comment (hrhino) [inline README.md]: this could also be some online IRC client link.

Comment (NeQuissimus) [reply README.md]: "online IRC client"... :no_good_man: 

`hrino gets slapped around a bit with a large trout`

Comment (hrhino) [reply README.md]: hey, I don't want it. I'm anticipating someone saying "but people won't be able to use it without a client installed!"

File: README.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ It defines a set of foundational type classes (e.g. `Functor`, `Monad`) and corr
...
+[![Gitter](http://badges.gitter.im/scalaz/scalaz.png)](https://gitter.im/scalaz/scalaz)
+[![Discord](https://img.shields.io/discord/455164441852641282.svg?label=Discord)](https://discord.gg/Rqb3y2a)
```
Comment (emilypi) [inline README.md]: _yiss_

**Pull request merged**